plugins/module_utils/spec.py

```diff
@@ -37,12 +38,14 @@
 class GenerateSpec(object):
     def __init__(
         self,
-        yang_file_path,
+        yang_content=None,
+        yang_file_path=None,
         search_path=None,
```

Comment:
The class object invocation in the spec lookup plugins should be updated accordingly. 
https://github.com/ansible-collections/community.yang/blob/main/plugins/lookup/spec.py#L229
```
        genspec_obj = GenerateSpec(
            yang_file_path=yang_file,
            search_path=search_path,
            doctype=doctype,
            keep_tmp_files=keep_tmp_files,
        )
```